docs/CHANGELOG.md

```diff
@@ -3,6 +3,9 @@ Version 3.40.0-eisop3 (November ??, 2023)
 
 **User-visible changes:**
 
+The nullness checker now issues redundant test against null warnings for switch statements when the expression is
+non-null and the case expression is null literal.
```

Comment:
This only happens when a command-line option is provided. How about pointing the reader to the section of the manual and/or listing the option?

Reply:
ok Thanks

> This only happens when a command-line option is provided. How about pointing the reader to the section of the manual and/or listing the option?

Thanks! Addressed!